### PR TITLE
SPCTR-2611: FontLoader and CSSLoader changes for Inline-notice, lottie, Marketing-button, Modal

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

- Base - https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/2929

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Comment by adi3890** [comment]
Can you please add some description to this PR

File: src/blocks/inline-notice/styling.js [hunk 1]
```diff
@@ -6,9 +6,9 @@ import generateCSS from '@Controls/generateCSS';
 import generateCSSUnit from '@Controls/generateCSSUnit';
 import { getFallbackNumber } from '@Controls/getAttributeFallback';
 
-function styling( props ) {
+function styling( attributes, clientId, name, deviceType ) { // eslint-disable-line no-unused-vars
```
Comment (adi3890) [inline src/blocks/inline-notice/styling.js:9]: can we replace this eslint comment

File: src/blocks/lottie/styling.js [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 import generateCSS from '@Controls/generateCSS';
 
-function styling( props ) {
+function styling( attributes, clientId, name, deviceType ) { // eslint-disable-line no-unused-vars
```
Comment (adi3890) [inline src/blocks/lottie/styling.js:7]: check if we can remove this eslint comment

File: src/blocks/marketing-button/styling.js [hunk 3]
```diff
@@ -6,9 +6,9 @@ import generateCSS from '@Controls/generateCSS';
 import generateCSSUnit from '@Controls/generateCSSUnit';
 import { getFallbackNumber } from '@Controls/getAttributeFallback';
 
-function styling( props ) {
+function styling( attributes, clientId, name, deviceType ) { // eslint-disable-line no-unused-vars
```
Comment (adi3890) [inline src/blocks/marketing-button/styling.js:9]: please check for removal of eslint comments in this PR

**Review by rahulvarma722 — APPROVED** [review]

File: src/blocks/inline-notice/styling.js (hunk 1 above)
Comment (rajmohanreddy774) [reply src/blocks/inline-notice/styling.js:9]: I have confirmed with Abhijith regarding same after discussion with Rajkiran we have finalized to keep it as it is.

File: src/blocks/lottie/styling.js (hunk 2 above)
Comment (rajmohanreddy774) [reply src/blocks/lottie/styling.js:7]: we are keeping this wantedly.

File: src/blocks/marketing-button/styling.js (hunk 3 above)
Comment (rajmohanreddy774) [reply src/blocks/marketing-button/styling.js:9]: we are keeping this wantedly.

**Comment by rajmohanreddy774** [comment]
> Can you please add some description to this PR

Added.

File: src/blocks/inline-notice/styling.js (hunk 1 above)
Comment (rajmohanreddy774) [reply src/blocks/inline-notice/styling.js]: Removed the eslint comment

File: src/blocks/lottie/styling.js (hunk 2 above)
Comment (rajmohanreddy774) [reply src/blocks/lottie/styling.js]: Removed eslint comment

File: src/blocks/marketing-button/styling.js (hunk 3 above)
Comment (rajmohanreddy774) [reply src/blocks/marketing-button/styling.js]: Removed eslint comment

**Review by AbhijitNage123 — APPROVED** [review]

**Review by rahulvarma722 — APPROVED** [review]

**Pull request merged**